portfolio/src/main/webapp/index.html

```diff
@@ -18,18 +18,19 @@ <h1>Portfolio</h1>
         <h1>Andre Battle</h1>
         <p>Click continue to see content</p>
         <button class="button" onclick="showNext()">Continue</button>
-        <button class="button" onclick="getDatastoreComments()">Show Comments</button>
-        <form name="form1" action="/data" method="POST">
+        <button class="button" onclick="showComments()">Show Comments</button>
```

Comment:
I can't see comments anymore. I'm not sure if it's a problem with fetching them, or submitting them in the first place.

Reply:
It was probably just me repeatedly deploying the appengine and breaking it. I got sentiment analysis working, comments should be fine now!